src/ImageSharp/Image/Image{TColor}.cs

```diff
@@ -63,6 +63,28 @@ public Image(Stream stream, Configuration configuration = null)
             this.Load(stream);
         }
 
+#if !NO_FILE_IO
```

Comment:
I still think I prefer feature based directives rather than platform based ones, but saying that I would be willing to accept NETSTANDARDX_X based ones to.

I originally went with the positive HAS_FILE_IO on all targets except the 1.1. target but I changed my mind last minute.